MMA Removal Utility/SingleTenantSetupInstallation.md

```diff
@@ -0,0 +1,302 @@
+# **Steps to install single-tenant AzTS MMA Removal Utility**
+
+In this section, we will walk you through the steps of setting up AzTS MMA Removal Utility. This setup can take up to 30 minutes.
+
+Before initiating the setup please validate the prerequisites, download and extract deployment package using [Prerequisites](./Prerequisites.md).
```

Comment:
Before initiating the setup please validate the prerequisites, download and extract deployment package using the link **here**.

Reply:
Addressed in all places